Makefile

```diff
@@ -0,0 +1,26 @@
+PROJECT_NAME := skynet_ai_frontend
+IMAGE_LOCATION := .
```

Comment:
No point in having this variable if it's only used once (taking up valuable space! :p ) - Also where this is used, it is commented out :)

Reply:
Yep. I guess since we're using docker-compose we don't need it any more. Removed. 8f5c3c61f17d0cf4a48aac4d413b4da46fa21e21